src/components/News.vue

```diff
@@ -23,5 +23,10 @@ export default {
   },
 }
 </script>
-<style>
+<style scoped lang="stylus">
+@import '~variables'
+.max-width-center
```

Comment:
possible to put this class: .max-width-center 
into common.variables.styl ?
coz it is used at many places